### DISCUSSION TITLE
Difference between accessing whisper from git or OPENAI Key

### DISCUSSION THREAD

**Discussion by JanmeetIITrpr** [openai/whisper#1088] [General]
What is the difference if i directly access whisper from it's cloned library rather than open ai api with key. And Why is this free whereas that costs me per token?

**Comment by EtienneAb3d** [comment]
> And Why is this free whereas that costs me per token?

This is simple: if you are using your own Whisper deployment it's your machine cost, while with API access it's using OpenAI CPU/GPU resources. GPU calculation is quite expensive.
